README.md

```diff
@@ -2,197 +2,230 @@
 
-```sh
-cp (npm root -g)/vtex/plugins/fish/fish_prompt.fish ~/.config/fish/functions/
-fish
-```
+When you log in at an account, you can create your own workspaces to work on your ideias or see what is on air. To do so:
 
-PS: If you used `yarn` to install, you should replace `(npm root -g)` with `(yarn global dir)`.
+1. On an app directory, run `vtex link` and click on or copy the provided URL into your browser;
+2. Now on, VTEX Toolbelt will monitor your files for change and sync them with what you see automatically. 
 
----
+<details>
+<summary>Know more about local development with Toolbelt</summary>
 
-# Frequently Asked Questions
+Toolbelt will upload all your app files to the developer environment, print an URL for your use, and watch for any changes you make to the files, which will be automatically synchronized.
 
-## How do I login?
+</details>
 
-You can use `vtex login` to login with your VTEX ID credentials or `vtex logout` if you're already logged in and want to change credentials.
 
-When logging in, you will be asked for **3** things:
+# Wanna know something cool? You can customize your prompt
 
-- The `account` name of the store you wish to work on
-- Your VTEX ID `e-mail`
-- Your VTEX ID `password`
+You can configure your **terminal prompt** to display relevant information about your current context, like which account you're logged into and which `workspace` you are currently using.
 
-If you wish to work on another `account`, logout and login again with that `account`.
+Just like knowing which `git` branch you're currently in, having this info in your prompt you help you avoid mistakes and be faster when using VTEX IO.
 
-Note that `link` and `publish` implicitly checks if you're logged, and if you're not, it asks your credentials before proceeding.
+## How to do so? 
 
+#### `bash` users
 
-## How do I develop an app locally?
+If you are a bash user you can start customizing your command prompt by running the following command:
 
-To develop an app locally, open the directory where your VTEX app is and then type:
+```sh
+echo "source $(npm root -g)/vtex/plugins/bash/vtex.bash" >> ~/.bashrc
+echo "source $(npm root -g)/vtex/plugins/bash/prompt.bash" >> ~/.bashrc
+```
+
+> 😉 For fine grained control, use only the first script and add `__vtex_ps1` manually to your prompt.
+
+#### `fish` users
+
+If you want a pretty, ready-made Git + VTEX prompt you can copy `plugins/fish/fish_prompt.fish` to the  `~/.config/fish/functions` folder. To see the changes, simply restart your shell by typing `fish`.
 
 ```sh
-vtex link
+cp (npm root -g)/vtex/plugins/fish/fish_prompt.fish ~/.config/fish/functions/
+fish
 ```
 
-Toolbelt will upload all your app files to the developer environment, print an URL for your use and watch for any changes you make to the files, which will be automatically synchronized.
+> 💡 If you installed Toolbelt using `yarn`, you should replace `(npm root -g)` with `(yarn global dir)`.
```

Comment:
Makes sense to me. 

I didn't update the content of the snippets because it is out of my coverage area by now. We can do it in a next update, as I would like to ask for a developer to review this specific topic and edit considering the previous changes. (I don't want to abuse your and Napoli's goodwill and ask you to edit it by now hahah)